templates/partials/page-header.hbs

```diff
@@ -37,7 +37,12 @@
           >Locations</a>
         </li>
         <li class="grav-c-nav-menu__external-link">
-          <a class="grav-c-block-link" href="https://medium.com/buildit">Blog</a>
+          <a
+            class="grav-c-block-link"
+            href="https://medium.com/buildit">
+            <svg class="medium-logo" width="45" height="45" viewBox="0 0 45 45"><path d="M5 40V5h35v35H5zm8.56-12.627c0 .555-.027.687-.318 1.03l-2.457 2.985v.396h6.974v-.396l-2.456-2.985c-.291-.343-.344-.502-.344-1.03V18.42l6.127 13.364h.714l5.256-13.364v10.644c0 .29 0 .342-.185.528l-1.848 1.796v.396h9.19v-.396l-1.822-1.796c-.184-.186-.21-.238-.21-.528V15.937c0-.291.026-.344.21-.528l1.823-1.797v-.396h-6.471l-4.622 11.542-5.203-11.542h-6.79v.396l2.14 2.64c.239.292.291.37.291.768v10.353z"></path></svg>
```

Comment:
Should have a `role="img"` and `aria-label="Medium"` attributes on the `<svg>`. That'll make it semantically equivalent to an `<img>` and provide a text alternating. The whole link _should_ then be read out as "Medium Blog" by screen readers.